libs/product/src/reducers/index.spec.ts

```diff
@@ -11,7 +11,7 @@ import { DaffProduct } from "../models/product";
 describe('selectProductState', () => {
 
   let store: Store<fromProduct.State>;
-  const productFactory: DaffProductFactory = new DaffProductFactory();
+  const productUnionFactory: DaffProductUnionFactory = new DaffProductUnionFactory();
```

Comment:
Since the `union` is so broad, can we get away with only using the `DaffProductFactory` here?

Reply:
So we can get rid of the union factory entirely?

Reply:
Yes.

Reply:
removed ProductUnion factory